moto/__init__.py

```diff
@@ -145,6 +145,11 @@ def f(*args, **kwargs):
 mock_xray = lazy_load(".xray", "mock_xray")
 mock_xray_client = lazy_load(".xray", "mock_xray_client")
 mock_wafv2 = lazy_load(".wafv2", "mock_wafv2")
+mock_sdb = lazy_load(".sdb", "mock_sdb")
```

Comment:
This looks like a dodgy merge - I changed the ordering earlier to be alphabetical, so SDB and ElastiCache are already defined above

Reply:
My bad for `backend_index,py`. It should be fine now. I also fixed `__init__.py`